dev-libs/libusb-compat/libusb_compat-0.1.7.recipe

```diff
@@ -39,8 +39,7 @@ BUILD_REQUIRES="
 	devel:libusb_1.0$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	cmd:aclocal
```

Comment:
Keep aclocal as a dependency, also "cmd:libtool(ize)" should be $secondaryArchSuffix